docs/deployment/cli.md

```diff
@@ -338,4 +338,74 @@ specified in the `config/flink-config.yaml`.
 For more details on the commands and the available options, please refer to the Resource Provider-specific 
 pages of the documentation.
 
+### Submitting PyFlink Jobs
+
+Currently, users are able to submit a PyFlink job via the CLI. It does not require to specify the
+JAR file path or the entry main class, which is different from the Java job submission.
+
+<span class="label label-info">Note</span> When submitting Python job via `flink run`, Flink will run the command "python". Please run the following command to confirm that the python executable in current environment points to a supported Python version of 3.5+.
+{% highlight bash %}
+$ python --version
+# the version printed here must be 3.5+
+{% endhighlight %}
+
+The following commands show different PyFlink job submission use-cases:
+
+- Run a Python Table job:
```

Comment:
Thinking about it: Why do we call it Python Table job? Isn't PyFlink job the correct way of labelling it?
This applies to all the occurrences below as well.

Reply:
Yes, Using `PyFlink job` would be more general.